docs/shared/recipes/plugins/create-preset.md

```diff
@@ -0,0 +1,88 @@
+## Preset
+
+A Preset is a customization option which you provide when creating a new workspace. TS, Node, React are some internal presets that Nx provides by default.
+
+{% youtube
+src="https://www.youtube.com/embed/yGUrF0-uqaU"
+title="Develop a Nx Preset for your Nx Plugin"
+width="100%" /%}
+
+### Custom Preset
+
+At its core a preset is a generator, which we can create inside of a plugin.
```

Comment:
Add here:

"All first-party Nx presets are built into nx itself, but you can [Create Your Own Plugin](link) and create a generator with the magic name: "preset". Once you've [Published Your Plugin](link) on npm, you can now run. the `create-nx-workspace` command with the preset option set to the name of your published package.

For example, take

```terminal
npx create-nx-workspace --preset=qwik-nx
```

This command will create a new qwik preset based on [the published npm package: "qwik-nx](https://www.npmjs.com/package/qwik-nx). If we check this package's source code, we can see that it has a [generator named "preset"](https://github.com/qwikifiers/qwik-nx/tree/main/packages/qwik-nx/src/generators/preset)